doc/api/next_api_changes/removals/26962-IA.rst

```diff
@@ -0,0 +1,22 @@
+Removal change template
+~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
```suggestion
```

Please remove the template text, and use `~~~~` underlines for each of the headings.